docs/gitrepo-targets.md

```diff
@@ -161,6 +161,13 @@ Fleet will deploy the Helm chart with your customized `values.yaml` to the diffe
 * [Helm.ValuesFrom](/ref-crds#helmoptions)
 * [Helm.Values](/ref-crds#helmoptions)
 * [Helm.Version](/ref-crds#helmoptions)
+:::warning important information
```

Comment:
How will this warning block be rendered? Should we indent it to preserve the enclosing list? Can we test this?